README.md

```diff
@@ -1,7 +1,185 @@
-# Booster Release Scripts
+# Release Instructions
 
-The bulk of the work to release both community and product boosters is described here:
+## Prerequisites
 
-- [Release Instructions](./HOWTO_RELEASE.md)
+- Clone this directory into $BOOSTER_HOME (your choice)
+- Fetch all forked booster repositories _into_ $BOOSTER_HOME (Need to be children of $BOOSTER_HOME). Make sure that the remote from https://github.com/thorntail-examples/ is called **upstream**. You can run `./clone_here.sh` to do this.
 
-The work around catalog updates will follow soon.
+Your workspace should like like this:
+
+```
+$ tree -L 1 -d
+.
+├── cache
+├── cache-redhat
+├── circuit-breaker
+├── circuit-breaker-redhat
+├── configmap
+├── configmap-redhat
+├── health-check
+├── health-check-redhat
+├── messaging-work-queue
+├── rest-http
+├── rest-http-crud
+├── rest-http-crud-redhat
+├── rest-http-redhat
+├── rest-http-secured
+└── rest-http-secured-redhat
+```
```

Comment:
May be add the istio-tracing in this tree ?

Reply:
I didn't add it to the `clone-here.sh` script yet :-)

Reply:
Sure, I guess it can be done when the `-redhat` version of this example is added